acme.sh

```diff
@@ -273,7 +304,8 @@ _debug() {
     _syslog "$SYSLOG_DEBUG" "$@"
   fi
   if [ "${DEBUG:-$DEBUG_LEVEL_NONE}" -ge "$DEBUG_LEVEL_1" ]; then
-    _printargs "$@" >&2
+    _bash_debug=$(__debug_bash_helper)
```

Comment:
just remove these.

Reply:
The reason I did it that was is to that when someone selects "--debug 3" then all the debug lines will get the filename, function name and line no. If we do it the other way then the "debug 1" and "debug 2" lines will never have the filename, function name, and line no even when debug 3 is used.